### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by marta-lokhova**

### CODE REVIEW

File: doc/missions.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+Perform recent catchup to Public Network. Runs core `catchup current/1001` command  under the hood.
+
+## MissionHistoryPubnetCompleteCatchup
+
+Perform full catchup to Public Network. Runs core `catchup current/max` command  under the hood.
+
+## MissionHistoryPubnetParallelCatchup
+
+Perform full catchup to Public Network, but in parallel to improve performance. Runs a mix of minimum and recent catchup commands under the hood.
```
Comment (sisuresh) [inline doc/missions.md:49]: Parallel catchup runs in batches of 16320 ledgers at the moment. It doesn't do minimum catchup. This is from the logs - 
```
2024-02-17 19:22:41.205 +00:00 [INF] Running 3151 jobs (128-way parallel) of 250 checkpoints each, to catch up to ledger 50415103 starting from 0
2024-02-17 19:22:41.205 +00:00 [INF] Printing catchup ranges
2024-02-17 19:22:41.225 +00:00 [INF] [|"catchup"; "15103/16320"|]
2024-02-17 19:22:41.225 +00:00 [INF] [|"catchup"; "31103/16320"|]
2024-02-17 19:22:41.225 +00:00 [INF] [|"catchup"; "47103/16320"|]
2024-02-17 19:22:41.225 +00:00 [INF] [|"catchup"; "63103/16320"|]
2024-02-17 19:22:41.225 +00:00 [INF] [|"catchup"; "79103/16320"|]
2024-02-17 19:22:41.225 +00:00 [INF] [|"catchup"; "95103/16320"|]
```

File: doc/missions.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+Perform recent catchup to Test Network. Runs core `catchup current/1001` command  under the hood.
+
+## MissionHistoryTestnetCompleteCatchup
+
+Perform full catchup to Test Network. Runs core `catchup current/max` command  under the hood.
+
+## MissionHistoryTestnetParallelCatchup
+
+Perform full catchup to Test Network, but in parallel to improve performance. Runs a mix of minimum and recent catchup commands under the hood.
```
Comment (sisuresh) [inline doc/missions.md:73]: Same comment as above

File: doc/missions.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+Complete extended payment load generation while triggering surge pricing with mixed version images on a small network of nodes. Tests both old image majority and new image majority.
+
+## MissionHistoryGenerateAndCatchup
+
+Test catchup to generated ledger state for each protocol version.
+
+## MissionHistoryPubnetMinimumCatchup
+
+Perform minimum catchup to Public Network. Runs core `catchup current/0` command  under the hood.
```
Comment (bboston7) [inline doc/missions.md:37]: Tiny nitpick: There are two spaces between "command" and "under" here and in a few other places in this file. Seems to just be a copy/paste error.

**Review by bboston7 — APPROVED** [review]
Thanks Marta, these new docs are helpful! I just found a tiny typo that has been copy/pasted a few times in this file.

File: doc/missions.md (hunk 1 above)
Comment (marta-lokhova) [reply doc/missions.md]: thanks, I clarified the description

File: doc/missions.md (hunk 3 above)
Comment (marta-lokhova) [reply doc/missions.md]: good catch, fixed.

**Pull request merged**